cpp/unreal_plugins/SimulationController/Source/SimulationController/SimulationController.cpp

```diff
@@ -294,9 +295,7 @@ void SimulationController::bindFunctionsToRpcServer()
         FGenericPlatformMisc::RequestExit(immediate_shutdown);
     });
 
-    rpc_server_->bindAsync("ping", []() -> std::string {
-        return "SimulationController received a call to ping()...";
-    });
```

Comment:
Can you please restore the old formatting for consistency with the other lambdas in this part of the code.